### ISSUE TITLE
Regarding loss convergence?

### ISSUE THREAD

**Issue by Mishra1995** [IIGROUP/MANIQA#31]
Hello authors,

Thanks for open-sourcing your code and models! It really helps in reproducing the results.

My query is regarding the loss convergence. I tried to train the model on KonIQ dataset before you modified the code and it seems the network is overfitting a lot with a huge gap between training and validation loss. I even tried to change the loss function and did normalisation  but it didn't factor.

Any suggestions regarding the same? Can you also share the loss plots to verify the convergence criteria?

Thanks!

**Comment by TianheWu** [comment]
When we train the MANIQA on Koniq10k dataset, we resize the image into 224x224. The model will reach the score in our checkpoints report. The loss value will converge.

**Comment by Mishra1995** [comment]
![MANIQA_origina](https://user-images.githubusercontent.com/107555279/225857001-59aa0059-e217-4834-9055-4068336b1592.png)


This is the loss plot I am getting after running the current code on KonIQ dataset for 63 epochs without changing any parameters. Is there anything else that needs to be done?

**Comment by TianheWu** [comment]
You can test your model's ability in your test set.
